python/docs/overview.rst

```diff
@@ -0,0 +1,229 @@
+.. _overview:
+
+Overview
+========
+
+Interpret-Community extends the [Interpret](https://github.com/interpretml/interpret) repository and incorporates further community developed and experimental interpretability techniques and functionalities that are designed to enable interpretability for real world scenarios.
+Interpret-Community enables adding new experimental techniques (or functionalities) and performing comparative analysis to evaluate them.
+
+Interpret-Community 
+
+1. Actively incorporates innovative experimental interpretability techniques and allows for further expansion by researchers and data scientists
+2. Applies optimizations to make it possible to run interpretability techniques on real-world datasets at scale
+3. Provides improvements such as the capability to "reverse the feature engineering pipeline" to provide model insights in terms of the original raw features rather than engineered features
+4. Provides interactive and exploratory visualizations to empower data scientists to gain meaningful insight into their data
+
+
+Getting Started
+===============
+
+The package can be installed from `pypi <https://pypi.org/project/interpret-community/>`_ with:
+
+   .. code-block:: bash
+
+      pip install interpret-community
+
+
+You can use Anaconda to simplify package and environment management.
+
+To setup on your local machine:
+
+.. raw:: html
+
+    <style>
+        #inner {
+            margin-left:50px; 
+            margin-right:-50px;
+        }
+    </style>
+    <details><summary><strong><em>1. Set up Environment</em></strong></summary>
+
+        a. Install Anaconda with Python >= 3.6
+        <br/>
+        <a href="https://conda.io/projects/conda/en/latest/user-guide/install/index.html">Miniconda</a> is a quick way to get started.
+        <br/>
+        <br/>
+        b. Create conda environment named interp and install packages
+        <br/>
+
+        <blockquote>
+            <div>
+                <div class="highlight-bash notranslate">
+                    <pre>conda create --name interp python=3.6 anaconda</pre>
```

Comment:
Still using 3.6? Maybe upgrade? It's rather old by now and removed by packages like sklearn

Reply:
I think we want to keep it until downstream projects that depend on this package fully deprecate 3.6 (eg azureml-sdk).  However, I do agree it might be a good idea to update the main docs to reference 3.7 now.